src/libraries/System.Private.CoreLib/src/System/IO/Strategies/Net5CompatFileStreamStrategy.Windows.cs

```diff
@@ -36,7 +36,7 @@
 
 namespace System.IO.Strategies
 {
-    internal sealed partial class Net5CompatFileStreamStrategy : FileStreamStrategy, IFileStreamCompletionSourceStrategy
+    internal sealed partial class Net5CompatFileStreamStrategy : FileStreamStrategy
```

Comment:
Consider keeping Net5CompatFileStreamStrategy "as-is" since we don't want to make any changes on legacy code unless they are meant to fix new issues.

Reply:
> we don't want to make any changes on legacy code unless they are meant to fix new issues.

@Jozkee you are right, but in this case, @carlossanlop is removing a bridge (`IFileStreamCompletionSourceStrategy`) that I've added in .NET 6 so it's something that we want